File: os2skoledata-azure-ad-sync/src/main/java/dk/digitalidentity/service/model/Institution.java
package dk.digitalidentity.service.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Map;

@Getter
@Setter
public class Institution {
	private long databaseId;
	@JsonProperty("number")
	private String institutionNumber;
	@JsonProperty("name")
	private String institutionName;
	private String abbreviation;
	private String allAzureSecurityGroupId;
	private String studentAzureSecurityGroupId;
	private String employeeAzureSecurityGroupId;
	private boolean locked;
	private String employeeAzureTeamId;
	private String teamAdminUsername;
	private Map<String, String> azureIdentifierMappings;
}

File: os2skoledata-ad-sync/Services/OS2skoledata/Model/Institution.cs
using Newtonsoft.Json;
using System;

namespace os2skoledata_ad_sync.Services.OS2skoledata.Model
{
    public class Institution
    {
        public long DatabaseId { get; set; }
        [JsonProperty("number")]
        public string InstitutionNumber { get; set; }
        [JsonProperty("name")]
        public string InstitutionName { get; set; }
        public string Abbreviation { get; set; }
        public InstitutionType Type { get; set; }
        public bool Locked { get; set; }
    }

    public enum InstitutionType
    {
        SCHOOL,
        DAYCARE,
        MUNICIPALITY,
        FU
    }
}